services/explorer/src/api.rs

```diff
@@ -529,6 +529,22 @@ impl ExplorerApi {
         Ok(CallStatusResponse { status })
     }
 
+    fn test_endpoint(
```

Comment:
Nit: It could make sense to test stuff in the `api` test in testkit; it's specifically designed for this purpose 🙃